jest-common/src/main/java/io/searchbox/indices/script/DeleteIndexedScript.java

```diff
@@ -0,0 +1,64 @@
+package io.searchbox.indices.script;
+
+import io.searchbox.action.AbstractAction;
+import io.searchbox.action.GenericResultAbstractAction;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+import static io.searchbox.indices.script.ScriptLanguage.GROOVY;
+
+public class DeleteIndexedScript extends GenericResultAbstractAction {
+
+  private final String scriptName;
+  private final ScriptLanguage scriptLanguage;
+
+  public DeleteIndexedScript(Builder builder) {
+    super(builder);
+    this.scriptName = builder.scriptName;
+    this.scriptLanguage = builder.scriptLanguage;
+    setURI(buildURI());
+  }
+
+  protected String buildURI() {
+    try {
+      return super.buildURI() + "/_scripts/" + scriptLanguage.pathParameterName + "/" + URLEncoder.encode(scriptName, CHARSET);
+    } catch (UnsupportedEncodingException e) {
+      throw new RuntimeException(e);
```

Comment:
> we could remove the encoding part.
> This would be consistent with the other code.

We are (supposed to be) url-encoding everything user can pass into URL because well handling the HTTP communication is kinda jest's responsibility (otoh ES server should be the one deciding and replying if it is valid as an index/type/scriptname etc.), if you found a place where we are not doing that please let me know so I can fix it. 
My comment was about re-throwing it as `RuntimeException`; in other cases of `UnsupportedEncodingException` we just log it as an error and move on.